### PR TITLE
Feature/card menu community

### PR DESCRIPTION
**Pull request by Zoeshwa**
useUser from sigin or signup  & edit a community

### CODE REVIEW

File: apps/ui/src/Communities/CommunitiesFeed.tsx [hunk 1]
```diff
@@ -49,16 +56,40 @@ const CommunitiesFeed = ({ communities }: CommunitiesFeedProps) => {
...
   const handleNewCommunity = (newCommunity: Community) => {
+    setUserCommunitiesStatus((prev) => ({
+      ...prev,
+      [newCommunity.id]: UserStatus.MANAGER,
+    }));
     setAllCommunitiesDisplay((prev) => [newCommunity, ...prev]);
   };
 
+  const handleUpdateCommunity = (communityUpdated: Community) => {
+    setAllCommunitiesDisplay((prev) =>
```
Comment (Kovler10) [inline apps/ui/src/Communities/CommunitiesFeed.tsx:82]: can we have the communities grouped by their id for faster updating?

File: apps/ui/src/ProtectedRoute.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Navigate, Outlet } from 'react-router-dom';
+import { useUser } from './hooks/Users/useUser';
+
+const ProtectedRoute = () => {
+  const { user } = useUser();
+
+  if (!user) {
```
Comment (Kovler10) [inline apps/ui/src/ProtectedRoute.tsx:7]: use ternary condition here (`!user ? <> : <>`)

**Review by Kovler10 — CHANGES_REQUESTED** [review]
Cool! Left some comments

File: apps/ui/src/themes/default/consts.ts [hunk 3]
```diff
@@ -61,13 +61,33 @@ export const SPACING = {
 } as const;
 
 export const TEXT = {
-  REGISTER: "Don't have an account? Register Now", 
-  FORGOT_PASSWORD: "Forgot password?",
-  SIGNIN: "Already have an account? Sign in",
-  LOGIN: "Login",
-  CONTINUE: "Continue",
+  REGISTER: "Don't have an account? Register Now",
```
[the remark this answers is not in the document]
Comment (Zoeshwa) [reply apps/ui/src/themes/default/consts.ts:64]: the format you told me to use is changing ' to " when save - you want me to turn the format off?

File: apps/ui/src/hooks/Communities/useGetAllCommunities.tsx [hunk 4]
```diff
@@ -18,20 +18,23 @@ const picturesUrl = [
 
 const communities: Community[] = [
   {
+    id: '1',
```
[the remark this answers is not in the document]
Comment (Zoeshwa) [reply apps/ui/src/hooks/Communities/useGetAllCommunities.tsx:21]: its just a dummy the backend will decide what id contains

**Review by Kovler10 — APPROVED** [review]

**Pull request merged**